crm_lead_lost_reason/wizard/lost_reason.py

```diff
@@ -28,10 +28,17 @@ class CrmLeadLost(models.TransientModel):
     _name = 'crm.lead.lost'
     _description = __doc__
 
+    def _default_reason(self):
+        active_id = self._context.get('active_id', [])
```

Comment:
you can remove the second argument of your call to get(): it defaults to `None`, which is `False`-ish